packages/webdriverio/src/middlewares.js

```diff
@@ -45,7 +58,19 @@ export const elementErrorHandler = (fn) => (commandName, commandFn) => {
             }).apply(this)
         }
 
-        return fn(commandName, commandFn).apply(this, args)
+        try {
+            return fn(commandName, commandFn).apply(this, args)
+        } catch(error) {
+            if (error.message.includes("stale element reference")) {
+                refetchElement(this).then(element => {
```

Comment:
same as above

Reply:
We aren't returning the updated element.. we are returning the result of the command with the updated context.

Reply:
Done.